src/SFA.DAS.EmployerAccounts.Web/Helpers/HtmlHelperExtensions.cs

```diff
@@ -22,13 +24,47 @@ public static bool IsAuthorized(this HtmlHelper htmlHelper, string featureType)
             return isAuthorized;
         }
 
+        public static string ReturnToHomePageButtonHref(this HtmlHelper htmlHelper, string accountId)
+        {
+            bool isTier2User = htmlHelper.ViewContext.RequestContext.HttpContext.User?.IsInRole(Tier2User) ?? false;
+            bool isAccountIdSet = accountId != null;
+
+            return isTier2User && isAccountIdSet ? $"accounts/{accountId}/teams/view" : isAccountIdSet ? $"accounts/{accountId}/teams": "/";
+        }
+
+        public static string ReturnToHomePageButtonText(this HtmlHelper htmlHelper, string accountId)
+        {
+            bool isTier2User = htmlHelper.ViewContext.RequestContext.HttpContext.User?.IsInRole(Tier2User) ?? false;
+            bool isAccountIdSet = accountId != null;
+
+            return isTier2User && isAccountIdSet ? "Return to your team" : isAccountIdSet ? "Go back to the account home page" : "Go back to the service home page";
+        }
+
+        public static string ReturnToHomePageLinkHref(this HtmlHelper htmlHelper, string accountId)
+        {
+            bool isTier2User = htmlHelper.ViewContext.RequestContext.HttpContext.User?.IsInRole(Tier2User) ?? false;
+            bool isAccountIdSet = accountId != null;
+
+            return  isTier2User && isAccountIdSet ? $"accounts/{accountId}/teams/view" : "/";
+        }
+
+        public static string ReturnToHomePageLinkHrefText(this HtmlHelper htmlHelper, string accountId)
```

Comment:
Better to be ReturnToHomePageLinkText perhaps?

Reply:
Updated